inst/htmljs/ldavis.js

```diff
@@ -425,63 +439,63 @@ LDAvis = function(to_select, json_file) {
             .text("Intertopic Distance Map (via multidimensional scaling)")
             .attr("x", mdswidth/2 + margin.left)
             .attr("y", 30)
-	    .style("font-size", "16px")
-	    .style("text-anchor", "middle");
+            .style("font-size", "16px")
+            .style("text-anchor", "middle");
 
         // establish layout and vars for bar chart
         var barDefault2 = lamData.filter(function(d) {
-            return d.Category == "Default"
+            return d.Category == "Default";
         });
 
         var y = d3.scale.ordinal()
-            .domain(barDefault2.map(function(d) {
-                return d.Term;
-            }))
-            .rangeRoundBands([0, barheight], 0.15);
+                .domain(barDefault2.map(function(d) {
+                    return d.Term;
+                }))
+                .rangeRoundBands([0, barheight], 0.15);
         var x = d3.scale.linear()
-            .domain([0, d3.max(barDefault2, function(d) {
-                return d.Total;
-            })])
-            .range([0, barwidth])
-            .nice();
+                .domain([1, d3.max(barDefault2, function(d) {
+                    return d.Total;
```

Comment:
I think the lower bound of the domain should remain at 0; otherwise, we run into the problem reported in #33